terraform/main.tf

```diff
@@ -0,0 +1,76 @@
+########################################################################################################################
+# Configure Google Cloud Provider
+########################################################################################################################
+
+terraform {
+  backend "remote" {
+    workspaces {
+      prefix = "observatory-"
```

Comment:
Won't this be deployed from a different workspace such as `academic-observatory-api-`?

Reply:
I've changed this to use the 'cloud' configuration instead like you commented yesterday. I've set it up so that it will use a tag 'academic-observatory-api', this is instead of the prefix.